### DISCUSSION TITLE
[ais-menu-select] Ability to select multiple items from the select box

### DISCUSSION THREAD

**Discussion by techEgab** [algolia/instantsearch#6324] [Q&A]
Hey ninjas, Is there a way to have a multiple selection in the ais menu select component?
![image](https://github.com/user-attachments/assets/cec9576e-4a2e-4e40-9a6d-bcce010bf291)

**Comment by Haroenv** [comment]
You want to create your own RefinementListSelect that uses `connectRefinementList` and `<select multiple>` (menu is one selection, refinement list multiple)

**Reply by techEgab** [reply]
I think this will be challenging when using state mapping using routing, in my case the value of the select input will be `Object[]`.

**Reply by Haroenv** [reply]
refinement list works perfectly fine with routing, where it's an array of strings. Do you have a sandbox that shows it not working?

**Reply by techEgab** [reply]
Still working on the sandbox
